FileIO/GmshIO/GMSHNoMeshDensity.h

```diff
@@ -31,12 +31,16 @@ class GMSHNoMeshDensity: public FileIO::GMSH::GMSHMeshDensityStrategy {
 		(void)(vec);
 	}
 
-	double getMeshDensityAtPoint(GeoLib::Point const*const pnt) const
+	double getMeshDensityAtPoint(GeoLib::Point const*const) const
+	{
+		return 0.0;
```

Comment:
Good point. The implementation was required since the virtual base class was extented. Today I discovered that the class `GMSHNoMeshDensity` is not used. I will delete it.